inventory.md

```diff
@@ -0,0 +1,53 @@
+# Inventory API
+
+## Service description
+The Inventory service keeps track of the stock level for variants within products. The API clients can list current stock levels and update balance for products. The service automatically decreases the stock when a purchase is made within the Zettle POS application.
+
+### Base URL
+https://inventory.izettle.com
+
+### API documentation
+https://inventory.izettle.com/swagger
+
+## How it works
+
+The service uses a concept of transactions where stock is moved between so called `Locations`.
+Locations are created automatically when tracking is enabled on any product. Currently existing locations and the natural flow of stock balance are explained in the following.
+
+![Stock balance flow from SUPPLIER to STORE and either BIN or SOLD](/inventory-api/images/Location_Flow.png)
+
+`SUPPLIER` is an abstraction of a supplier and holds infinite stock.
+
+`STORE` is the location of the current stock for a variant.
+
+`BIN` and `SOLD` are abstractions for when a product is sold or discarded.
+
+It is important to note that you cannot set a stock balance for a variant directly.
+To set an absolute value of a variant you need to calculate the change needed to reach that stock value.
+
+#### Example 1:
+`PUT /organizations/{organizationUuid}/inventory`
+
+You have 30 stock in `STORE` and want to set that to 20. To do this you move 10 stock to the `SOLD`, `BIN` or `SUPPLIER` location, depending on the reason for the stock change. Note that you always move positive numbers.
+
+```
+  "productUuid": <uuid>
+  "variantUuid": <uuid>,
+  "fromLocationUuid": <STORE_location_uuid>,
+  "toLocationUuid": “<SOLD_location_uuid>,
+  "change": 10
+```
```

Comment:
```json  
"productUuid": <uuid>
  "variantUuid": <uuid>,
  "fromLocationUuid": <STORE_location_uuid>,
  "toLocationUuid": “<SOLD_location_uuid>,
  "change": 10
```

Reply:
since the request body is json, should we maybe add the curly braces around the properties? it would still only be a fragment of the full request body though.

```json
{
  "productUuid": <uuid>,
  "variantUuid": <uuid>,
  "fromLocationUuid": <STORE_location_uuid>,
  "toLocationUuid": “<SOLD_location_uuid>,
  "change": 10
}
```

Reply:
for reference, here's a full request body (i copied the example from swagger):
```json
{
  "changes": [
    {
      "productUuid": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
      "variantUuid": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
      "fromLocationUuid": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
      "toLocationUuid": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
      "change": ">0"
    }
  ],
  "returnBalanceForLocationUuid": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "externalUuid": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
}
```
it could be nice to show the full request body, but i think all the extra fluff around the change distracts more than it helps.